indico/modules/attachments/controllers/event_package.py

```diff
@@ -227,21 +245,21 @@ def _prepare_form(self):
         return form
 
     def _load_session_data(self):
-        return [(session.getId(), escape(to_unicode(session.getTitle()))) for session in self._conf.getSessionList()]
+        return [(sess.id, escape(to_unicode(sess.title))) for sess in self.event_new.sessions]
 
     def _load_contribution_data(self):
         def _format_contrib(contrib):
-            if contrib.getSession() is None:
-                return to_unicode(contrib.getTitle())
+            if contrib.session is None:
+                return to_unicode(contrib.title)
```

Comment:
all the `to_unicode` calls are not needed anymore when you deal with new objects